src/components/DirectoryItemSelector/directory-item-selector.tsx

```diff
@@ -77,7 +77,7 @@ function updatedTree(prevRoots: any[], prevMap: any, nodeId: UUID | null, childr
                 return pn;
             }
             if (pn.parentUuid !== nodeId) {
-                console.warn(`reparent ${pn.parentUuid} -> ${nodeId}`);
+                console.warn(`reparent ${pn?.parentUuid} -> ${nodeId}`);
```

Comment:
I don't understant how it fix null exception. 

if you optional chainning is the fix, that mean that "pn" is undefined.
and line 79 if "pn" is undefined you will get the same null exception.

line 68 if "pn" is undefined we return a value, so line 80 pn couldn't be undefined.

Reply:
I apologize for the confusion. The title of the pull request was a mistake, and the bug is not resolved yet. I will investigate further to address the null exception issue properly.